W3ChampionsStatisticService/Hubs/WebsiteBackendHub.cs

```diff
@@ -276,6 +276,33 @@ await Clients.Caller.SendAsync(
         }
     }
 
+    public async Task BlockPlayer(string battleTag)
+    {
+        var currentUser = _connections.GetUser(Context.ConnectionId)?.BattleTag;
+        if (currentUser == null)
+        {
+            return;
+        }
+        try
+        {
+            var friendList = await _friendCommandHandler.LoadFriendList(currentUser);
+            CanBlock(friendList, battleTag);
+            friendList.BlockedBattleTags.Add(battleTag);
+            await _friendCommandHandler.UpsertFriendList(friendList);
+            await Clients.Caller.SendAsync(
+                FriendResponseType.FriendResponseData.ToString(),
+                friendList,
+                null,
+                null,
+                $"Player {battleTag} blocked!"
+            );
+        }
+        catch (Exception ex)
+        {
+            await Clients.Caller.SendAsync(FriendResponseType.FriendResponseMessage.ToString(), ex.Message);
```

Comment:
Do we need to log this exception?

Reply:
Not questions I feel equipped to answer tbh, it's the same code pattern as `UnblockFriendRequestsFromPlayer` directly below this fn 🤷‍♂️

Reply:
I'd recommend you add a `using Serilog` at the top and then you do `Log.Error` (or `Log.Exception` if that's a thing) before you do the await.